plugin.go

```diff
@@ -1,7 +1,7 @@
 package main
 
 import (
-	"github.com/appleboy/drone-git-push/repo"
```

Comment:
Thanks @bradrydzewski, that is very helpful.  I guess I need to clone my fork into `$GOPATH/src/github.com/appleboy/drone-git-push`